### PR TITLE
Add plausible analytics

### PR DESCRIPTION
**Pull request by Integrayshaun**
## What changed?
- [x] Add Plausible plugin with patch for Netlify proxying
- [x] Add redirects for Netlify proxying
- [x] Add `patch-package` for patching plugin
- [x] Alphabetize package scripts (Because I love us and want to be nice to us)

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -111,6 +111,16 @@ module.exports = {
       },
     },
     `gatsby-plugin-sitemap`,
+    ...(!isDeployPreview
```
Comment (kylegach) [inline gatsby-config.js:114]: This would result in the plugin running during local development. Will Plausible have any issue with that?

**Review by kylegach — APPROVED** [review]
👍

File: gatsby-config.js (hunk 1 above)
Comment (jonniebigodes) [reply gatsby-config.js:114]: I ran it locally, and checking the dev tools, nothing is being shown regarding Plausible, so it seems we're in the clear. @Integrayshaun can you confirm if anything is being tracked

Comment (Integrayshaun) [reply gatsby-config.js]: Plausible wont, but I'm sure that we don't want localhost records in our analytics. Fixed

**Pull request merged**